forms/serializers/form.py

```diff
@@ -203,7 +192,7 @@ def to_representation(self, instance):
     def filter_subsections(data):
         sections = []
         for sec in data["sections"]:
-            if sec["parent"] is None:
+            if sec["parent_id"] is None:
```

Comment:
Is there possibility that there is no parent_id, as then there would be exception.

Same kind of rows are bottom also

Reply:
to_representation has this "sections" and section.parent_id always, because it generates the output data from the whole instance even if user has patched only one field etc. I'll check the rest of the cases though.